tasks/go.py

```diff
@@ -391,6 +411,40 @@ def is_excluded(pkg):
                         print("Skipping {} ({}) excluded in .wwhrd.yml".format(package, license))
                     else:
                         licenses.append("core,\"{}\",{}".format(package, license))
+
+    # Additional Licenses
+    for pkg in additional:
+        if pkg.startswith(GITHUB_DOMAIN):
+            url = "https://{}/master/".format(pkg.replace(GITHUB_DOMAIN, GITHUB_RAW_DOMAIN))
+        else:
+            url = "https://{}".format(pkg)
+
+        for filename in LICENSE_FILES:
+
+            url = urljoin(url, filename)
+            try:
+                resp = requests.get(url)
+                resp.raise_for_status()
+
+                with tempfile.TemporaryDirectory() as tempdir:
+                    with open(os.path.join(tempdir, filename), 'w') as lfp:
+                        lfp.write(resp.text)
+                        lfp.flush()
```

Comment:
It looks like `go-license-detector` might be able to retrieve the license itself from a Github URL, see https://github.com/go-enry/go-license-detector#usage maybe we should do that? It would simplify the code quite a bit and avoid dealing with installing `requests`.

Reply:
It clones the entire repository making it terribly slow, I already tried that.

Reply:
I opened  go-enry/go-license-detector#7 to improve it a bit. It shaves off a few seconds (~30s vs ~25s on my machine) so it's still pretty slow :(